2-Browsers/Week1/homework/ex5-catWalk/index.js

```diff
@@ -17,6 +17,31 @@
 -----------------------------------------------------------------------------*/
 function catWalk() {
   // TODO complete this function
+  const img = document.querySelector('img');
+  const img1 = 'http://www.anniemation.com/clip_art/images/cat-walk.gif';
```

Comment:
Good practice is to give more descriptive variable names, so this would be catWalkImageSrc, and below it would be catCelebratingImageSrc

Reply:
I will update these right away!